src/hotspot/cpu/s390/vm_version_s390.hpp

```diff
@@ -44,6 +44,8 @@ class VM_Version: public Abstract_VM_Version {
 //   z196: 2010-08
 //   ec12: 2012-09
 //   z13:  2015-03
+//   z14:  2017-09
+//   z15:  2019-09
```

Comment:
How does this relate to the table in the .cpp file? I'd prefer to have such kind of information consolidated at one place.